docs/sphinx/_toc.yml.in

```diff
@@ -19,3 +19,13 @@ subtrees:
   - caption: About
     entries:
     - file: license
+  - caption: Sample
```

Comment:
I'd change this to Samples

Reply:
thanks for the suggestion, I had renamed it.